ooDesign/frankensteinWithConstructors/solutionM/build.gradle

```diff
@@ -0,0 +1,16 @@
+plugins {
+    id 'java'
+}

```

Comment:
please add checkstyle in to the gradle file so that the example solution is also an example of code that adheres to checkstyle.